src/absil/ilread.fs

```diff
@@ -1029,377 +1032,373 @@ let count c =
 #endif
         
 
-let seekReadUInt16Adv ctxt (addr: byref<int>) =  
```

Comment:
@dsyme As JavaScript lang does not support byref parameters unless they're wrapped in objects, there will be no performance advantage in JS. I've opened a Fable feature request for the byref support, but perhaps we can use struct tuples here, unless you prefer #ifdefs to retain the byref for dotnet?